relayer/app/metrics.go

```diff
@@ -75,4 +75,11 @@ var (
 		Name:      "head_height",
 		Help:      "The latest height of different types of head blocks on a specific chain",
 	}, []string{"chain", "type"})
+
+	attestedHeight = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "relayer",
+		Subsystem: "monitor",
+		Name:      "halo_attested_height",
+		Help:      "The latest halo attested height of a specific chain",
+	}, []string{"chain"})
```

Comment:
Maybe this can be a good candidate for an exemplar.

Reply:
mmm, yeah, let me think what we can add